### PR TITLE
[FIX] base: access right on partner.merge.wizard are too restricted

### PR DESCRIPTION
**Pull request by tfr-odoo**
Before this commit only admin setting user can merge contact which is a bit to restrictive
Contract Creation group seems more adequate

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/base/security/ir.model.access.csv [hunk 1]
```diff
@@ -118,5 +118,5 @@
 "access_base_language_export","access.base.language.export","model_base_language_export","base.group_user",1,1,1,0
 "access_base_update_translations","access.base.update.translations","model_base_update_translations","base.group_system",1,1,1,0
 "access_base_partner_merge_line","access.base.partner.merge.line","model_base_partner_merge_line","base.group_system",1,1,1,0
```
Comment (mart-e) [inline odoo/addons/base/security/ir.model.access.csv:120]: this one too maybe?

Comment (tfr-odoo) [reply odoo/addons/base/security/ir.model.access.csv]: indeed

**Comment by tfr-odoo** [comment]
@mart-e r+ ?

**Comment by mart-e** [comment]
@robodoo @tfr-odoo r+

**Pull request closed without merging**